nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/state/flow/flow.reducer.ts

```diff
@@ -400,20 +415,26 @@ export const flowReducer = createReducer(
         ...state,
         operationCollapsed
     })),
-    on(startComponentSuccess, stopComponentSuccess, (state, { response }) => {
-        return produce(state, (draftState) => {
-            const collection: any[] | null = getComponentCollection(draftState, response.type);
+    on(
+        startComponentSuccess,
+        stopComponentSuccess,
+        enableComponentSuccess,
+        disableComponentSuccess,
```

Comment:
We are missing a reducer for `enableProcessGroupSuccess` and `disableProcessGroupSuccess` that sets the flowState saving back to false.

If you enable/disable the current process group (right click on the canvas) then try to edit one of the components in that PG, the apply button is a spinner as it thinks it is still "saving"
<img width="772" alt="Screenshot 2024-04-23 at 10 17 08 AM" src="https://github.com/apache/nifi/assets/713866/8866a3e9-d911-47d9-a825-9b9c4564711c">

However, if you were to enable/disable a child pg this isn't an issue as the child pg is reloaded and causes `loadChildProcessGroupSuccess` to fire which ultimately sets the saving state back to false.